about.html

```diff
@@ -0,0 +1,105 @@
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-facebook-f"></i>
+        <ul class="flist">
+          <li><a href="">English</a></li>
+          <li><a href="">My page</a></li>
+          <li><a href="">logout</a></li>
+        </ul>
+      </div>
+      <div class="bar" id="bar">
+        <i class="fa-solid fa-bars fa-2x"></i>
+      </div>
+      <div class="hide" id="close">
+        <i class="fa-solid fa-xmark "></i>
+      </div>
+      <nav class="lnav">
+        <div class="logo">
+          <h1><a href="index.html">AYC</a></h1>
+        </div>
+        <div >
+          <ul class="barlist">
+            <li><a href="">About</a></li>
+            <li><a href="">Program</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Sponsor</a></li>
+            <li><a href="">News</a></li>
+            <li><button>AYC company</button></li>
+          </ul>
+      </div>
+      </nav>  
+    
+    </header>
+    <div class="hi" id="hi">
+      <h2>African Youth Confrance 2022</h2>
+      <p>As a National Resource Center for African Studies, the Harvard University Center for African Studies compiles Africa-related seminars, workshops, events, and conferences taking place in Cambridge, Boston, and Johannesburg, South Africa. We also include virtual Africa-related events and activities that are free and open to the public. The Center for African Studies will consider requests to host, coordinate, sponsor, or co-sponsor Africa-related events only if the event is supported by one of our faculty affiliates.</p>
+      <h3 class="date">July 27 ~ 28</h3>
+      <h3 class="place">@ University of Notre Dame.</h3>
+    </div>
+    </section>
+
+    <section>
+      <div class="info">
+        <p>please contact us through email for further information.</p>
+        <p class="email">gdureti@gmail.com</p>
+      </div>
+    </section>
+    <section>
+      <div class="past">
+        <p class="pastp">The past African youth confrance event</p>
+        <hr>
+      </div>
+      <div class="event">
+        <div>
+          <div class="past-event-overlay">
+            <h1 class="past-event-title">2020</h1>
+            <p class="past-event-par">Youth Confrance</p>
+          </div>
+          <img class="past-pic" src="images/past1.jpg" alt="">
+        </div>
+        <div>
+          <div class="past-event-overlay">
+            <h1 class="past-event-title">2021</h1>
+            <p class="past-event-par">Youth Confrance</p>
+          </div>
+          <img class="past-pic" src="images/past4.jpg" alt="">
+        </div>
+      </div>
+    </section>
+    <!--Parteners-->
+<section class="parteners">
+  <div>
+    <h2>Parteners</h2>
+    <hr>
+  </div>
+  <div class="image">
+    <img id="club" src="images/club.webp" alt="">
+    <img id="cole" src="images/cole.webp" alt="">
+    <img id="xender" src="images/Xender.webp" alt="">
+    <img id="Au" src="images/Au.jpg" alt="">
+  </div>
+</section>
```

Comment:
- [ ] Please remove the partners section from your about page to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170754558-ed6e0c16-1843-493d-adbe-4e9cd4029ff0.png)  | ![image](https://user-images.githubusercontent.com/68796232/170754276-63ac96c2-bd5f-494a-8cd5-e308d7bc52ea.png) |